packages/analytics/src/public-types.ts

```diff
@@ -288,4 +288,14 @@ export interface EventParams {
   page_path?: string;
   [key: string]: unknown;
 }
+
+/** Maps the applicable end user consent state. */
```

Comment:
Suggest: "Consent status settings for each consent type." or similar, @egilmorez may have suggestions. Also add `@public` tag.